airbyte-integrations/connectors/destination-mysql-strict-encrypt/Dockerfile

```diff
@@ -16,5 +16,5 @@ ENV APPLICATION destination-mysql-strict-encrypt
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.1.4
```

Comment:
the fact that it is not 0.1.5 is fishy

Reply:
Are we missing some commits somewhere?

Reply:
It seems like the PR from the changelog that said it bumped it to 0.1.5 didn't actually bump this connector: https://github.com/airbytehq/airbyte/pull/12820/files